setup.sh

```diff
@@ -25,6 +25,18 @@ pip install pycurl
 
 pip install mgmtsystem
 
+# setup test recorder
+wget https://pypi.python.org/packages/1e/8e/40c71faa24e19dab555eeb25d6c07efbc503e98b0344f0b4c3131f59947f/vnc2flv-20100207.tar.gz
+tar -zxvf vnc2flv-20100207.tar.gz
```

Comment:
Is there a python module that can be pip installed, instead of using a URL that will most likely change?